awx/ui_next/src/screens/Job/JobOutput/shared/jobOutputUtils.js

```diff
@@ -6,7 +6,7 @@ export default function getRowRangePageSize(startIndex, stopIndex) {
     page = startIndex + 1;
     pageSize = 1;
   } else if (stopIndex >= startIndex + 50) {
-    page = Math.ceil(startIndex / 50);
```

Comment:
This line of code generally did the right thing, unless `startIndex` is 0.

When `startIndex` is 0, `ceil` won't adjust it to `1`:

![Screenshot from 2021-03-31 12-15-37](https://user-images.githubusercontent.com/9753817/113179002-29ec7000-921d-11eb-9abe-79348fa60992.png)